v20.1/admin-ui-cluster-overview-page.md

```diff
@@ -3,88 +3,131 @@ title: Cluster Overview Page
+Nodes | Nodes are grouped by locality and displayed with their address. Click the address to view node statistics. Hover over a row and click **Logs** to see the node's log.
+Uptime | Amount of time the node has been running.
+Replicas | Number of replicas on the node or in the locality.
+Capacity Usage | Percentage of usable disk space occupied by CockroachDB data on the node or in the locality. See [Capacity metrics](#capacity-metrics).
+Memory Usage | Memory used by CockroachDB as a percentage of the total memory on the node or in the locality.
+CPUs | Number of vCPUs on the machine.
+Version | Build tag of the CockroachDB version installed on the node.
 
 ### Decommissioned Nodes
 
-Nodes that have been decommissioned for permanent removal from the cluster are listed in the **Decommissioned Nodes** table.
+Nodes that have recently been decommissioned for permanent removal from the cluster are listed in the table of **Recently Decommissioned Nodes**. You can see the full history of decommissioned nodes by clicking "View all decommissioned nodes".
 
-When you decommission a node, CockroachDB lets the node finish in-flight requests, rejects any new requests, and transfers all range replicas and range leases off the node so that it can be safely shut down. See [Remove Nodes](remove-nodes.html) for more information.
+<img src="{{ 'images/v20.1/admin-ui-decommissioned-nodes.png' | relative_url }}" alt="CockroachDB Admin UI node list" style="border:1px solid #eee;max-width:100%" />
+
+{{site.data.alerts.callout_info}}
+When you [decommission a node](remove-nodes.html), CockroachDB lets the node finish in-flight requests, rejects any new requests, and transfers all range replicas and range leases off the node so that it can be safely shut down.
+{{site.data.alerts.end}}
 
 ## Node Map (Enterprise)
 
-The **Node Map** is an [enterprise-only](enterprise-licensing.html) feature that gives you a visual representation of the geographical configuration of your cluster.
+The **Node Map** is an [enterprise](enterprise-licensing.html) feature that visualizes the geographical configuration of your cluster. It requires that [`--locality` flags have been defined](cockroach-start.html#locality) for your nodes.
+
+For guidance on enabling and configuring the node map, see [Enable the Node Map](enable-node-map.html).
 
 <img src="{{ 'images/v20.1/admin-ui-node-map.png' | relative_url }}" alt="CockroachDB Admin UI Summary Panel" style="border:1px solid #eee;max-width:90%" />
 
-The Node Map consists of the following components:
+The Node Map is populated with **locality components** and **node components**, using the latitude and longitude of each locality to position the components on the map.
+
+### Locality component
+
+The map shows the components for the highest-level locality tier (e.g., region). You can click on the **Node Count** of a locality component to view any lower-level localities (e.g., availability zone). 
```

Comment:
I think it would be helpful to start this section with "A locality component is ..." and describe what it is, and then launch into the further detail.